script/testing/junit/__main__.py

```diff
@@ -2,14 +2,151 @@
 
```

Comment:
@turingcompl33t So I think I actually misread this, I think this file was an intended change right?
I like the way you nuked constants.py and in general it looks good to me